client/main.js

```diff
@@ -97,7 +97,11 @@ main = (function(main, global) {
     });
 
 
-    $('#nav-chat-btn').on('click', toggleMainChat);
+    $('#nav-chat-btn').on('click', function(event) {
+      if (!event.ctrlKey && !event.metaKey) {
```

Comment:
@george-stepanek Ah you are correct. But `!(event.ctrlKey || event.metaKey)` IMHO is much more descriptive than `(!event.ctrlKey && !event.metaKey)` @hallaathrad What do you think?